### PR TITLE
Feature/add kenney assets tool

### PR DESCRIPTION
**Pull request by Stedders**
#2 

Added kenney tooling and yfinance.

### CODE REVIEW

File: coder_dojo_common/settings.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
```
Comment (jonathancychow) [inline coder_dojo_common/settings.py:1]: Why do we need this `settings.py`?

**Review by jonathancychow — DISMISSED** [review]
Nice work, I have not bothered to test the kenny cli, it's good to go if you are confidence that will work in both window and mac.

File: coder_dojo_common/kenney_assets.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+def get_assets(session: Session, base_url: str) -> Dict[str, str]:
+    """
+    Get all assets from the Kenney site.
+    """
+    page_number = 1
+    found_assets = {}
+    while True:
+        response = session.get(f"{base_url}/page:{page_number}")
+        if response.status_code == 404:
```
Comment (dannystaple) [inline coder_dojo_common/kenney_assets.py:18]: I'm assuming this is handling the paging? Probably worth a comment.

File: coder_dojo_common/settings.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+from pydantic import BaseSettings, validator
+
+
+class CoderDojoSettings(BaseSettings):
+    root_dir: Path = Path.home() / ".coder_dojo"
+
+    # Configuration
+    class Config:
+        """
+        Set env prefix, root values cane be overriden from environment variables prefixed with CODER_DOJO_
```
Comment (dannystaple) [inline coder_dojo_common/settings.py:12]: Typo `cane` should be `can`.

File: coder_dojo_common/models.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
+"""Data structures/models"""
+
+from __future__ import annotations
```
Comment (dannystaple) [inline coder_dojo_common/models.py:3]: I'm wondering if these should be in a kenney_assets package within coder_dojo_common - the cli, kenney_assets and models.py?

**Comment by dannystaple** [comment]
Couple of small comments - but downloaded and tested - `poetry run kenney-assets query`, and `poetry run kenney-assets install` worked a treat.

**Review by dannystaple — APPROVED** [review]

File: coder_dojo_common/settings.py (hunk 1 above)
Comment (Stedders) [reply coder_dojo_common/settings.py:1]: It acts as base settings for everything in the package - where we stored cached assets, etc...

**Comment by Stedders** [comment]
@dannystaple @jonathancychow  - could you resolve your comments if everything is fine or push back if you want me to change.

**Review by dannystaple — APPROVED** [review]

**Pull request merged**